src/Assets/index.js

```diff
@@ -12,16 +12,16 @@ import brand2 from "./images/brand-2.png"
 import brand3 from "./images/brand-3.png"
 import brand4 from "./images/brand-4.png"
 import brand5 from "./images/brand-5.png"
-import loginpic from "./images/login-pic.jpg"
-import loginpic2 from "./images/login-2.jpg"
+import login1 from "./images/login-1.jpg"
 import banner1 from "./images/banner-1.png"
 import banner2 from "./images/banner-2.png"
 import banner3 from "./images/banner-3.png"
```

Comment:
suggestion: instead of importing and exporting from index.js, you can directly export from this file like
export brand4 from "./images/brand-4.png"